### PR TITLE
desktop menu sidebar added, main sidebar, header, footer items adjusted

### PR DESCRIPTION
**Pull request by dev-catalyst**
Menu rearrangement: https://app.clickup.com/t/37wrym7

- DesktopMenu sidebar added
- Main sidebar items adjusted
- Docs added to footer
- Png & return to legacy buttons removed from header

### CODE REVIEW

File: src/layout/Sidebar/index.tsx [hunk 1]
```diff
@@ -1,19 +1,96 @@
...
+  ]
+
+  const pangolinLinks: OtherLink[] = [
+    {
+      link: ANALYTICS_PAGE,
+      icon: Charts,
+      title: t('header.charts'),
+      id: 'charts'
+    }
+  ]
```
Comment (SarjuHansaliya) [inline src/layout/Sidebar/index.tsx:93]: I think charts link should go into right desktop sidebar. please confirm with Abdullah

File: src/layout/Header/MobileMenu/index.tsx [hunk 2]
```diff
@@ -40,6 +45,98 @@ const MobileMenu: React.FC<Props> = ({ activeMobileMenu, handleMobileMenu }) =>
...
+      title: `Avalanche ${t('header.bridge')}`,
+      id: 'bridge'
+    },
+    {
+      link: 'https://satellite.axelar.network/',
+      icon: BridgeIcon,
+      title: `Satellite ${t('header.bridge')}`,
+      id: 'satellite-bridge'
+    }
+  ]
```
Comment (SarjuHansaliya) [inline src/layout/Header/MobileMenu/index.tsx:138]: we are repeating many of the things. we should have a way to reuse it.

for e.g. may be in one file ( `menuItems.ts` in Header folder ) you can keep separate objects like this 
```js
export const bridge = {
  link: "https://bridge.avax.network/",
  icon: BridgeIcon,
  title: `Avalanche ${t("header.bridge")}`,
  id: "bridge",
};

export const satelliteBridge = {
  link: 'https://satellite.axelar.network/',
  icon: BridgeIcon,
  title: `Satellite ${t('header.bridge')}`,
  id: 'satellite-bridge'
}
```
and then in this file you just use it
```js
const otherLinks = [bridge, satelliteBridge]
```

in that way there will not be any duplication for links and labels. 
this is just an idea, if you have any better idea i am up for it.

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/layout/Header/MobileMenu/index.tsx (hunk 2 above)
Comment (dev-catalyst) [reply src/layout/Header/MobileMenu/index.tsx]: I've moved out the common static data. But  have kept the _title_ and _isActive_ keys same due to use of hooks.

File: src/layout/Sidebar/index.tsx (hunk 1 above)
Comment (dev-catalyst) [reply src/layout/Sidebar/index.tsx]: Confirmed, it'll be on left.

File: src/layout/Header/DesktopMenu/index.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import { useWindowSize } from '../../../hooks/useWindowSize'
+import { LEGACY_PAGE, MENU_LINK } from '../../../constants'
+import { Button, useTranslation } from '@pangolindex/components'
+import { Sider, TopBar, LegacyButtonWrapper } from './styled'
+import { Scrollbars } from 'react-custom-scrollbars'
+import { MenuLinks } from '../../Sidebar/MenuLinks'
+import { MainLink, OtherLink } from '../../types'
+import { Vote } from '../../../components/Icons'
+import BridgeIcon from '../../../assets/svg/menu/bridge.svg'
+import GovernanceIcon from '../../../assets/svg/menu/governance.svg'
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/layout/Header/DesktopMenu/index.tsx]: Import paths are updated.

**Pull request closed without merging**